src/components/SQFormDialog/SQFormDialogInner.js

```diff
@@ -40,54 +40,96 @@ function SQFormDialogInner({
   saveButtonText,
   shouldRequireFieldUpdates = false,
   title,
-  muiGridProps
+  muiGridProps,
+  showCancelConfirmDialog,
+  cancelConfirmDialogProps
 }) {
   const actionsClasses = useActionsStyles();
   const {resetForm} = useSQFormContext();
 
-  const handleClose = () => {
+  const [
+    isDialogAlertModalOpen,
+    {openDialog: openDialogAlertModal, closeDialog: closeDialogAlertModal}
+  ] = useDialog();
+
+  const handleCancel = () => {
+    if (!showCancelConfirmDialog) {
+      resetForm();
+      onClose();
+
+      return;
```

Comment:
For short and sweet if/else scenarios that don't return a value, if/else block is generally more declarative.

Reply:
@SeanGroff are we not following internal coding standards for SQForm?